theme.config.jsx

```diff
@@ -0,0 +1,8 @@
+const themeConfig = {
```

Comment:
Could it be `.tsx` instead of `.jsx`

Reply:
no, the nextra says that this must be jsx